src/main.js

```diff
@@ -49,6 +49,12 @@ window.onload = () => {
             // Results from querying vector tiles
             vtQueryResults: { type: 'FeatureCollection', features: [] },
             countries: [],
+            worldviews: [
+                { code: 'cn', name: 'China' },
+                { code: 'in', name: 'India' },
+                { code: 'jp', name: 'Japan' },
+                { code: 'us', name: 'US (De facto)' }
```

Comment:
What are your thoughts about using the term 'Default' in place of 'De facto'?

Reply:
I have been mostly referring to terminologies used on [boundaries playground](https://demos.mapbox.com/vt_polygons/)  and on [api-docs](https://docs.mapbox.com/vector-tiles/reference/mapbox-boundaries-v3/#--polygon---worldview-text).

<img width="1573" alt="image" src="https://user-images.githubusercontent.com/17143802/88575593-c2fd3280-d061-11ea-8d0d-0ead2579bd20.png">